config/initializers/bugsnag.rb

```diff
@@ -1,10 +1,17 @@
+# frozen_string_literal: true
+
 Bugsnag.configure do |config|
   config.api_key = Rails.configuration.three_scale.bugsnag_api_key
   config.app_version = System::Deploy.info.revision
   config.notify_release_stages = %w[production]
   stages = Rails.configuration.three_scale.error_reporting_stages
   config.notify_release_stages = stages if stages.present?
 
-  # when WebHooks fails because the remote is down or similar - we don't mind
-  config.ignore_classes << ->(error) { WebHookWorker::ClientError === error }
+  ignore_error_names = ActionDispatch::ExceptionWrapper.rescue_responses.each_with_object([]) do |(key, value), classes|
```

Comment:
Why not just add all of them directly, because rails considers them to be rescued anyway